server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2602,7 +2613,7 @@ public void testShouldSelectPoolWhenAddonsMatchButOtherSysPurposeAttributesAndSo
 
         // Candidate pools:
         Product prodMCT1650 = createSysPurposeProduct(null, "random_role", "myaddon",
-            "random_sla", "random_usage");
+            "random_sla", "random_usage", null);
```

Comment:
Same here: This should be "random_service_type" instead of null, since the test expects all attributes other than addons to have some random value

Reply:
Done.